mage_ai/settings/__init__.py

```diff
@@ -31,6 +31,10 @@ def is_disable_pipeline_edit_access():
 REQUIRE_USER_AUTHENTICATION = \
     os.getenv('REQUIRE_USER_AUTHENTICATION', 'False').lower() in ('true', '1', 't')
 AUTHENTICATION_MODE = os.getenv('AUTHENTICATION_MODE', 'LOCAL')
+try:
+    MAGE_ACCESS_TOKEN_EXPIRY_TIME = int(os.getenv('MAGE_ACCESS_TOKEN_EXPIRY_TIME', 3600))  
+except ValueError:
+    MAGE_ACCESS_TOKEN_EXPIRY_TIME = 3600
```

Comment:
can we change the default to 30 days in seconds, so that it matches the previous logic?